rayon-core/src/registry.rs

```diff
@@ -224,11 +224,14 @@ impl Registry {
 
         let (workers, stealers): (Vec<_>, Vec<_>) = (0..n_threads)
             .map(|_| {
-                if breadth_first {
-                    deque::fifo()
+                let worker = if breadth_first {
+                    deque::Worker::new_fifo()
```

Comment:
`Worker` should be in scope, so I don't think we need `crossbeam_deque as deque` anymore.